code/modules/surgery/surgery_step.dm

```diff
@@ -85,6 +86,12 @@
 	fail_prob = min(max(0, modded_time - (time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)),99)//if modded_time > time * modifier, then fail_prob = modded_time - time*modifier. starts at 0, caps at 99
 	modded_time = min(modded_time, time * SURGERY_SLOWDOWN_CAP_MULTIPLIER)//also if that, then cap modded_time at time*modifier
 
+	// Skyrat Edit Addition - reward for doing surgery in surgery
+	if(istype(get_area(target), /area/medical/surgery) && (TRAIT_FASTMED in user.status_traits))
+		modded_time /= SURGERY_SPEEDUP_AREA
```

Comment:
...and use *= instead.

Reply:
How dare you make me following standard coding practices